app/clients/sms/firetext.py

```diff
@@ -57,8 +57,7 @@ def get_name(self):
     def record_outcome(self, success, response):
         status_code = response.status_code if response else 503
 
-        log_message = "API {} request {} on {} response status_code {}".format(
-            "POST",
+        log_message = "API POST request {} on {} response status_code {}".format(
```

Comment:
we should be able to use an %s with am if/else condition, any reason why we are not?

Reply:
%s should only be used inside logging statements.
https://docs.python.org/3.8/howto/logging.html#logging-variable-data

Reply:
sorry, got thrown off by this being a log_message. Ideally, an f-string should be used for consistency?